### ISSUE TITLE
Using @javax.validation.constraints.Size with only max contraint may result in an empty string

### ISSUE THREAD

**Issue by hfcipriano** [devopsfolks/podam#244]
```
Class A {
    @Size(max = 4)
    @NotEmpty()
    String b;
    get set...
}

A result = podam.manufacturePojo(A.class);
```

In some invokes the b attribute has the value "" (Empty) what is invalid

Sorry for the previously equivocated question

**Comment by daivanov** [comment]
In first case you set property to null explicitely outside of Podam. Then b is obviously null.

**Comment by hfcipriano** [comment]
If i use just only NotEmpty or Size  it's works perfectly but i need to define a String not empty with at most 4 characteres

**Comment by daivanov** [comment]
Thus, @Size(min = 1, max = 4) is not an option?

**Comment by hfcipriano** [comment]
Not because i would to differentiate messages when the user input a empty string and when user input max lenght string..

@Size(max = 4, message = "max.limitation")
@NotEmpty(message="not.empty")

**Comment by hfcipriano** [comment]
I commented into changeset

> What i'm saying is that sometimes the generated value is correct another times the value is "" (Empty string)
>

**Comment by daivanov** [comment]
Released in podam 7.1.0
